scripts/upgradeConfig.ts

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+import { readdir, readFile, writeFile } from 'fs-extra';
+import simpleGit from 'simple-git';
+
+/**
+ * Script: upgradeConfigs.ts
+ * Run with: ts-node scripts/upgradeConfig.ts
+ * ------------------------------------------
+ * Upgrades the references to \@solid/community-server in
+ * package.json (lsd prefix) and all JSON-LD config files.
+ * This script is run alongside standard-version
+ * after the version bump is done in package.json but before the
+ * release has been committed.
+ */
+
+/**
+ * Helper function to escape search strings for use in Regular Expressions
+ * @param  string - the search string to be escaped
+ * @returns Promise with the escaped string
+ */
+async function escapeString(string: string): Promise<string> {
+  return string.replace(/[$()*+.?[\\\]^{|}]/gu, '\\$&');
+}
+
+interface NpmPackage {
+  version: string;
+  name: string;
+}
+/**
+ * Reads and return the package.json file to extract version and name
+ * @returns Promise with the NpmPackage
+ */
+async function getNpmPackage(): Promise<NpmPackage> {
+  return JSON.parse(await readFile('package.json', 'utf8'));
+}
```

Comment:
We're going to have a utility function for this soon: https://github.com/CommunitySolidServer/CommunitySolidServer/pull/1286/files#diff-b2c17cdaffd9f24e5b97bb04476db87f2b0eaff6bd2e89ff0aea097b0cb06a39R274-R280

Unfortunately that is the 5.0.0 branch. So perhaps just copy that to this branch as well to the same location and then it will get solved when merging main into 5.0.0 later.